docker-compose.yml

```diff
@@ -18,12 +18,6 @@ services:
     volumes: [ ".:/work" ]
     command: "bundle exec jekyll serve --livereload --host=0.0.0.0"
     ports: [ "4000:4000", "35729:35729" ]
-  preview-nuxt:
```

Comment:
Astro has a native `astro preview` command that serves our static files created during the build process. We don't need to do this ourselves anymore 🥳